ast/src/gen/generic.rs

```diff
@@ -1,816 +1,915 @@
 // File automatically generated by ast/asdl_rs.py.
 
+use crate::text_size::TextRange;
 #[derive(Clone, Debug, PartialEq)]
-pub struct ModModule<U = ()> {
-    pub body: Vec<Stmt<U>>,
-    pub type_ignores: Vec<TypeIgnore>,
+pub struct ModModule<R = TextRange> {
+    pub body: Vec<Stmt<R>>,
+    pub type_ignores: Vec<TypeIgnore<R>>,
+    #[cfg(feature = "all-nodes-with-ranges")]
+    pub range: R,
+    #[cfg(not(feature = "all-nodes-with-ranges"))]
+    pub range: crate::EmptyRange<R>,
```

Comment:
by looking sum types, range seems to come first to unconditionally generate `fn range()` to same target code

Reply:
I moved the `field` to the top but Rust doesn't guarantee fixed field ordering (without `repr(c)`). So it may be the first field, or it may not...